### PR TITLE
Update content_how_to_get_compensated_for_work.md

### PR DESCRIPTION
**Pull request by vsterling**
Updating the live-old version of How to Get Compensated to include updated formatting and information.

### CODE REVIEW

File: en/content_how_to_get_compensated_for_work.md [hunk 1]
```diff
@@ -1,47 +1,59 @@
 # How to Get Compensated for Work
 
-## Where do I Find Work?
+The [Smart Contract Research Forum](smartcontractresearch.org) (SCRF) has created “How to Get Compensated for Work” for individuals to learn about SCRF’s payment system.
```
Comment (ghettodev) [inline en/content_how_to_get_compensated_for_work.md:3]: Vertical quotes, please.

Also, not sure what the utility is of a sentence that says that scrf has created the doc that people are reading. Everything is explained in the next paragraph.

File: en/content_how_to_get_compensated_for_work.md [hunk 2]
```diff
@@ -1,47 +1,59 @@
...
-* conference travel
-* charitable donations to an endowment in lieu
-* credits for compute resources, archive nodes, etc
-* supporting a blockchain-related organization or event of your choice
+SCRF officially pays invoices once per month at the end of the month. SCRF may process another round of payments mid-month, in certain situations. **All invoices are due two full business days before the end of the month**. Any billable hours forgotten or worked after the invoice was submitted can be added to the next invoice with a description. SCRF uses Request Network for all payments.
 
-Pay varies according to the complexity of the assignment and the time involved. Contributors who commit to longer-term engagements and have produced a good body of work for the forum are encouraged to seek additional and more lucrative responsibilities.
+* Use an Ethereum address. (SCRF recommends [Metamask](https://metamask.io/) but any exchange wallet is acceptable when transferring funds to a fiat wallet right away.)
+* Create an account on [Request Network](https://app.request.finance/login).
+* Choose USD for invoice currency.
```
Comment (ghettodev) [inline en/content_how_to_get_compensated_for_work.md:24]: I believe this is out of date, please confirm with @vincenzospaghetti about the currencies.

File: en/content_how_to_get_compensated_for_work.md [hunk 3]
```diff
@@ -1,47 +1,59 @@
...
+* Select DAI stablecoin as the payment cryptocurrency.
+* Add a new client using the email address invoice@scrf.io.
+  * Smart Contract Research Forum
+  * 1250 Broadway
+  * 36th Fl
+  * 10001 New York
```
Comment (ghettodev) [inline en/content_how_to_get_compensated_for_work.md:30]: It's a bit weird to have an address as list items. a code or a quote block would make more sense. 

```
 Smart Contract Research Forum
 1250 Broadway
 36th Fl
 10001 New York
```
or

> Smart Contract Research Forum
> 1250 Broadway
> 36th Fl
> 10001 New York

File: en/content_how_to_get_compensated_for_work.md [hunk 4]
```diff
@@ -1,47 +1,59 @@
...
-## Invoicing and Payment System
+* Funding a project
+* Conference travel
+* Charitable donations to an [endowment](https://thegivingblock.com/donate/)
+* Credits for compute resources, archive nodes, etc.
+* Direct support of a blockchain-related organization or event of choice.
+
+## Notes
+
+For questions, concerns, or to make alternative arrangements, contact SCRF’s Operations Team in a [Discord chat](https://discord.com/channels/784234332617048065/784234333111451670).
```
Comment (ghettodev) [inline en/content_how_to_get_compensated_for_work.md:49]: Please replace "Discord Chat" with "Chat"

File: en/content_how_to_get_compensated_for_work.md [hunk 5]
```diff
@@ -1,47 +1,59 @@
...
+
+* Please note that participation in community calls, events, and cohorts are not billable.
+* Invoices are due two full business days before the end of each month. Make sure to keep an eye on the Discord chat for any announcements, such as holidays, that may impact the invoice due date.
```
Comment (ghettodev) [inline en/content_how_to_get_compensated_for_work.md:52]: This is out of date. It should be 5 days. Please coordinate with @vincenzospaghetti about the current billing cycle.

**Review by ghettodev — CHANGES_REQUESTED** [review]

File: en/content_how_to_get_compensated_for_work.md (hunk 5 above)
Comment (vsterling) [reply en/content_how_to_get_compensated_for_work.md:52]: Hello, this is an example.

**Review by vsterling — COMMENTED** [review]
sample

**Comment by vsterling** [comment]
I will resubmit this PR when all changes and further updates are made to the How To Get Compensated doc.

**Pull request closed without merging**